src/gen.sh

```diff
@@ -24,7 +24,7 @@ mkdir codelists
 for f in ${codelists[*]}; do
     cp IATI-Codelists-NonEmbedded/xml/$f.xml codelists
 done
-unofficial_codelists=("HumanitarianGlobalClusters" "RegionM49")
```

Comment:
Hmm we could scrap the loop here, and just do this:
```
cp Unofficial-Codelists/xml/* codelists
```

Reply:
(We have the loop for IATI-Codelists-NonEmbedded because we only want to pick specific codelists. For Unofficial-Codelists we just want all of them.)